app/models/AppResult.java

```diff
@@ -78,7 +79,7 @@
   }
 
   public static String getSearchFields() {
-    return Utils.commaSeparated(AppResult.TABLE.NAME, AppResult.TABLE.USERNAME, AppResult.TABLE.JOB_TYPE,
+    return Utils.commaSeparated(AppResult.TABLE.NAME, AppResult.TABLE.USERNAME, TABLE.QUEUE_NAME, AppResult.TABLE.JOB_TYPE,
```

Comment:
I added it here, considering queuename as a search field in form. Will remove it.